### ISSUE TITLE
It worked well, but the results were not good enough,did I do something wrong?

### ISSUE THREAD

**Issue by phper5** [milesial/Pytorch-UNet#170]
after 5 epoch i got an image like this:
![output](https://user-images.githubusercontent.com/29189650/80898290-3d086e80-8d34-11ea-9d5e-bad76aaf82d3.jpg)
i downloaded datasets from [https://www.kaggle.com/c/carvana-image-masking-challenge/data](https://www.kaggle.com/c/carvana-image-masking-challenge/data)
unzip train_hq.zip, put them into data/imgs/  
unzip train_masks.zip,put them into data/masks/ 

<pre>
 data
      imags
          0cdf5b5d0ce1_01.jpg
         0cdf5b5d0ce1_02.jpg
        ....
    masks
        0cdf5b5d0ce1_01_mask.gif
        0cdf5b5d0ce1_02_mask.gif
     ....
</pre>

and then i just run `python train.py`

when it finished 5 epoch . i try ` python predict.py -i data/imgs/0cdf5b5d0ce1_02.jpg -o /tmp/output.jpg -m=checkpoints/CP_epoch5.pth `  I got the picture above

now i run it with 10 epoch again. but i am not sure weather I do something wrong  or just not enough training

thanks a lots

**Comment by phper5** [comment]
![2020-05-03_14-45](https://user-images.githubusercontent.com/29189650/80907860-ebb8a900-8d4c-11ea-878b-c19fc2924c20.png)
![2020-05-03_14-47](https://user-images.githubusercontent.com/29189650/80907879-1a368400-8d4d-11ea-8dbe-6755ed5bb844.png)

**Comment by milesial** [comment]
Hi, I think you should try with defferent hyperparameters, mainly a lower LR.
On my machine I train it with `python train.py -e 10 -b 1 -l 0.000002 -s 0.3`, but you can adjust the scaling (`-s`) and the batch size (`-b`) to your machine.

**Comment by phper5** [comment]
thanks mileial
i will try a lower LR and check the results after a few hours

**Comment by phper5** [comment]
thanks it is much better this time
